firebase-vscode/webviews/components/ui/PanelSection.tsx

```diff
@@ -0,0 +1,45 @@
+import { VSCodeDivider } from "@vscode/webview-ui-toolkit/react";
+import React, { ReactNode, useState } from "react";
+import { Icon } from "./Icon";
+import { Spacer } from "./Spacer";
+import { Heading } from "./Text";
+import cn from "classnames";
+import styles from "./PanelSection.scss";
+
+export function PanelSection({
+  title,
+  children,
+  isLast,
+}: {
+  title?: ReactNode;
+  children: ReactNode;
+  isLast?: boolean;
+}) {
+  let [isExpanded, setExpanded] = useState(true);
+  if (!title) {
+    isExpanded = true;
```

Comment:
This is confusing -- consider just changing the condition on Line 35 instead.

Reply:
Yeah I don't actually understand the purpose of the conditional because line 18 defaults it to true anyway. Removed the conditional. We do need the state because it's toggleable by the user.

Reply:
Without the extra logic, say if a user closes a panel but later the panel title is changed to empty, they will be left in a state where they can no longer see anything at all (since there's no title and no expand button)